createrepo_mod/createrepo_mod.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+"""
+A small wrapper around `createrepo_c` and `modifyrepo_c` to provide an easy tool
+for generating module repositories.
+
+This is supposed to be only a temporary solution, in the future we would like to
+have the modularity support implemented in `createrepo_c` itself. See
+
+https://bugzilla.redhat.com/show_bug.cgi?id=1816753
+
+Please see the official Fedora Modularity documentation for the reference of how
+module repositories should be created
+
+https://docs.fedoraproject.org/en-US/modularity/hosting-modules/
+"""
+
+
+import os
+import sys
+import subprocess
+import argparse
+
+
+CREATEREPO_BIN = "/usr/bin/createrepo_c"
+MODIFYREPO_BIN = "/usr/bin/modifyrepo_c"
```

Comment:
I wouldn't hard-code the paths. It's useful to rely on PATH to allow running them from a different location during testing or development.

Reply:
Ok, ok.

Reply:
Fixed